components/data-table/__examples__/advanced-single-select-fixed-header.jsx

```diff
@@ -124,7 +130,8 @@ class Example extends React.Component {
 						onRowChange={this.handleChanged}
 						onSort={this.handleSort}
 						selection={this.state.selection}
-						selectRows="radio"
+						disabledSelection={this.state.disabledSelection}
+						selectRows="checkbox"
```

Comment:
This is the single select example for consumers. Please keep `radio` or choose another example.

Reply:
reverted the changes.  I added a disabled checkbox in fixed-header example, which is showing checkbox currently